python/pyspark/mllib/evaluation.py

```diff
@@ -183,7 +183,7 @@ class MulticlassMetrics(JavaModelWrapper):
     0.66...
     >>> metrics.recall()
     0.66...
-    >>> metrics.accuracy()
+    >>> metrics.accuracy
```

Comment:
While we're here, maybe we should remove the deprecated calls to recall() and precision() above.